contracts/Market.sol

```diff
@@ -735,7 +741,7 @@ contract Market is
    * @return The amount of the fee for Nori.
    */
   function calculateNoriFee(uint256 amount) external view returns (uint256) {
-    return (amount * _noriFeePercentage) / 100;
+    return (amount * _priceMultiple * _noriFeePercentage) / 100 / 100;
```

Comment:
Can we use muldiv here to optimize gas?

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/Math.sol#L55

Reply:
Also, I find it a bit confusing that price multiple is in n*10^2 but the fee percentage is not. That might just be me though so happy to leave this unaddressed if so.

Reply:
They're both represented the same way here by my read -- in hundredths of the utility token.

Reply:
I think it does make sense because both of them are technically scaled by 100.  For example, a fee percentage of 15% is the value .15 which we represent as 15.  If the price multiple were also .15 (which I presume it never would be because that's less than one, but just for example) it would be represented as 15.  Anything that needs decimal precision has to be scaled by the number of digits of precision needed, and both of these need two digits and are scaled by 100.

Reply:
oops answered this before refreshing and seeing your comments scott. sounds like we're on the same page -- not trying to overexplain you jaycen haha